Header header
float32[] properties_data
float32[] objects_data
float32[] agents_data
